checklists.xml

```diff
@@ -304,47 +304,87 @@
+              ttsname="Takeoff data" ttsaction="Set"
+              audio-prompt="cklist-audio/QTTakeoff.opus"
+              audio-response="cklist-audio/QTSet.opus"/>
         <item name="10. AVIONICS/CHARTS" action="SET"
-              ttsname="Avionics, charts" ttsaction="Set"/>
+              ttsname="Avionics, charts" ttsaction="Set"
+              audio-prompt="cklist-audio/QTAvionicsCharts.opus"
+              audio-response="cklist-audio/QTSet.opus"/>
         <item name="11. FLT INSTRUMENTS" action="CHECKED/SET"
-              ttsname="Flight instruments" ttsaction="Checked, set"/>
+              ttsname="Flight instruments" ttsaction="Checked, set"
+              audio-prompt="cklist-audio/QTFLTInst.opus"
+              audio-response="cklist-audio/QTChkSet.opus"/>
         <item name="12. ANTI-SKID" action="ARMED/CHECK"
-              ttsname="Anti-skid" ttsaction="Armed, checked"/>
+              ttsname="Anti-skid" ttsaction="Armed, checked"
+              audio-prompt="cklist-audio/QTAntiSkid.opus"
+              audio-response="cklist-audio/QTArmChk.opus"/>
         <item name="13. PARKING BRAKE" action="SET"
-              ttsname="Parking brake" ttsaction="Set"/>
+              ttsname="Parking brake" ttsaction="Set"
+              audio-prompt="cklist-audio/QTParkingBrake.opus"
+              audio-response="cklist-audio/QTSet.opus"/>
         <item name="14. PASSENGER SIGNS" action="ON/AUTO"
-              ttsname="Passenger signs" ttsaction="On, auto"/>
+              ttsname="Passenger signs" ttsaction="On, auto"
+              audio-prompt="cklist-audio/QTPaxSigns.opus"
+              audio-response="cklist-audio/QTOnAuto.opus"/>
         <item name="15. DEPARTURE BRIEFING" action="COMPLETE"
-              ttsname="Departure briefing" ttsaction="Complete"/>
+              ttsname="Departure briefing" ttsaction="Complete"
+              audio-prompt="cklist-audio/QTDepBrief.opus"
+              audio-response="cklist-audio/QTComplete.opus"/>
         <item name="16. EMERGENCY PROCEDURES" action="REVIEW"
-              ttsname="Emergency procedures" ttsaction="Reviewed"/>
+              ttsname="Emergency procedures" ttsaction="Reviewed"
+              audio-prompt="cklist-audio/QTEmerProc.opus"
+              audio-response="cklist-audio/QTRev.opus"/>
         <item name="17. PINS/CHOCKS" action="REMOVED"
-              ttsname="Pins, chocks" ttsaction="Removed"/>
+              ttsname="Pins, chocks" ttsaction="Removed"
+              audio-prompt="cklist-audio/QTPinsChocks.opus"
+              audio-response="cklist-audio/QTRemoved.opus"/>
         <item name="18. QUICK TURN CHECKLIST" action="COMPLETE"
-              ttsname="Quick turn checklist complete" ttsautocomplete="true"/>
+              ttsname="Quick turn checklist complete" ttsautocomplete="true"
+              audio-prompt="cklist-audio/QTCComp.opus"
+              audio-response="cklist-audio/eng_start_next.opus"/>
```

Comment:
```suggestion
              audio-prompt="cklist-audio/QTCComp.opus"
              auto-adv="none"/>
```

`audio-response=` attribute value is not used here at all. Also add `auto-adv="none"` for consistency with Toto's checklists.